org.gnuhealth.mygnuhealth.yaml

```diff
@@ -0,0 +1,368 @@
+id: org.gnuhealth.mygnuhealth
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+
+command: mygnuhealth
+
+finish-args:
+  - --socket=wayland
```

Comment:
with all these x11 specific utils installed, does it work properly with wayland?